control-plane/src/values.js

```diff
@@ -0,0 +1,23 @@
+// ALL THE VALUES THAT ARE COMMONLY USED THROUGHOUT THE FILES
+const collection = process.env.NODE_ENV === 'production' ? 'jobs' : 'jobs-test'
+
+export default {
+    currentLevelsOne: 0,
+    currentLevelsTwo: 0,
+    jobStatus: {
```

Comment:
Are these for icon mappings? If so I recommend to be specific on the variable name --> jobStatusIcons

Reply:
Done